scripts/battery_remain.sh

```diff
@@ -18,15 +18,32 @@ print_battery_remain() {
 		pmset_battery_remaining_time
 	elif command_exists "upower"; then
 		battery=$(upower -e | grep battery | head -1)
-		upower -i $battery | grep remain | awk '{print $4}'
+		if is_chrome; then
+			if battery_discharging; then
+				upower -i $battery | grep 'time to empty' | awk '{printf "- %s %s left", $4, $5}'
+			else
+				upower -i $battery | grep 'time to full' | awk '{printf "- %s %s till full", $4, $5}'
+			fi
+		else
+			upower -i $battery | grep 'time to empty' | awk '{printf "- %s %s left", $4, $5}'
```

Comment:
In #30, @edouard-lopez implemented support for UPower 0.99.4. Your code will not support versions of `upower` where the `remain` is used instead of `time to empty`. Could you add the backwards compatibility? Thanks!